tests/Oryx.Integration.Tests/Node/NodeDynamicInstallationTest.cs

```diff
@@ -168,54 +170,7 @@ public async Task CanBuildAndRunApp_UsingScriptCommand_WithBullseyeBasedImages(s
                 .AddCommand(DefaultStartupFilePath)
                 .ToString();
 
-            await EndToEndTestHelper.BuildRunAndAssertAppAsync(
```

Comment:
why are we removing this?

Reply:
I didn't actually remove the test, there are 2 exactly same tests CanBuildAndRunApp_UsingScriptCommand_WithBullseyeBasedImages
CanBuildAndRunNode20App_UsingScriptCommand_WithBullseyeBasedImages
with different versions. so clubbed them into single test and passed version as parameter